src/utils/Authorizations.ts

```diff
@@ -0,0 +1,115 @@
+import { useStore } from '@/store/store'
+import { AccountTypes, AuthorizationRoles, AuthorizedActions } from '@/enums'
+
+/**
+ * Whether the specified action is authorized for the current user.
+ * Ultimately we'll just check if the auth roles includes the specified action.
+ * @returns True or False
+ */
+export function IsAuthorized (action: AuthorizedActions): boolean {
+  switch (true) {
+    case isBusinessRegistryStaff(): return BusinessRegistryStaffRoles.includes(action)
+    case isMaximusStaff(): return MaximusStaffRoles.includes(action)
+    case isContactCentreStaff(): return ContactCentreStaffRoles.includes(action)
+    case isSbcFieldOfficeStaff(): return SbcFieldOfficeStaffRoles.includes(action)
+    default: return DefaultRoles.includes(action)
+  }
+}
```

Comment:
Ultimately, I think "IsAuthorized()" could be a store getter or common function in a mixin.